README.md

```diff
@@ -5,65 +5,63 @@
 [![Download Status](https://img.shields.io/npm/dm/node-vault.svg?style=flat-square)](https://www.npmjs.com/package/node-vault)
 [![Dependency Status](https://img.shields.io/david/kr1sp1n/node-vault.svg?style=flat-square)](https://david-dm.org/kr1sp1n/node-vault)
 
-A client for the HTTP API of HashiCorp's [Vault] written for Node.js.
+NodeJS API client for HashiCorp's [Vault]
 
 
-## install
-make sure to use node.js version >= 6
+## Install
+make sure to use node.js version >= 8
 
+    # yarn
+    yarn add node vault
+    
+    # npm
     npm install node-vault
 
 
-## test
+## Usage
 
-Run tests inside docker to do also nice integration testing:
-
-    docker-compose up --force-recreate test
-
-This will create containers for vault, postgres and running the tests inside
-docker.
-
-
-## usage
+**[All features](features.md)**
 
-### init and unseal
+### Init and unseal
 
 ```javascript
-var options = {
+const options = {
   apiVersion: 'v1', // default
   endpoint: 'http://127.0.0.1:8200', // default
-  token: '1234' // optional client token; can be fetched after valid initialization of the server
-};
+  // VAULT_ADDR environment variable is also available
+  token: '<vault token>' // client authentication token
+  // VAULT_TOKEN environment variable is also available
+}
 
 // get new instance of the client
-var vault = require("node-vault")(options);
+const vault = require('node-vault')(options);
 
 // init vault server
 vault.init({ secret_shares: 1, secret_threshold: 1 })
-.then( (result) => {
```

Comment:
When we only support Node 8 we should rewrite all example with `async/await`

Reply:
Agreed

Reply:
[Done](https://github.com/kr1sp1n/node-vault/commit/3175f44bf6523413f2c8488bcd11adac8caed427)

Reply:
[And done in README](https://github.com/DaniGuardiola/node-vault/commit/c6a23d7587361c61ee82ca8bae47325fc427a3b5)